app/views/error/errorSummaryOrgTemplate.scala.html

```diff
@@ -30,7 +30,7 @@ <h2 id="error-summary-heading" class="h3-heading">@Messages("validation.summary.
 
         <ul>
             @form.errors.map { error =>
-                <li role="tooltip" id='@error.key.replaceAll("""[\.\[\]]""", "_")Error'>
+                <li role="tooltip" id='@error.key.replaceAll("""[\.\[\]]""", "_")Error' data-journey='There is a problem - view:@gaActionPage:@error.message'>
```

Comment:
How come there's a data-journey on the `<li>` here, rather than a data-journey-click on the `<a>` inside it? Is that right?